server/middleware/mentor/index.js

```diff
@@ -0,0 +1,99 @@
+const logger = require('simple-node-logger').createSimpleLogger();
+const {
+  HttpStatusCodes: { StatusCodes },
+  setLogDetails,
+} = require('../../utils/constants');
+const mongoUtil = require('../../utils/mongoUtil');
+
+const resObj = (statusCode, message, data = {}) => ({
+  statusCode,
+  message,
+  data,
+});
+
+const queryMentors = async ({ email, role_id }) => {
+  let data;
+  let statusCode;
+  let message;
+  let collectionObj;
+
+  logger.isInfo(
+    setLogDetails(
+      'mentorMiddleware.getMentors',
+      'Fetching metors data',
+      `User - ${email}`
+    )
+  );
+
+  try {
+    if (role_id !== 1 && role_id !== 0) throw Error('Bad request');
+
+    collectionObj = await mongoUtil.fetchCollection(
+      process.env.USERS_COLLECTION
+    );
+
+    data = await collectionObj
+      .find(
+        { role_id: 0 },
+        {
+          projection: {
+            _id: 0,
+            firstName: 1,
+            lastName: 1,
+            title: 1,
+            skills: 1,
+            bio: 1,
+          },
+        }
+      )
+      .toArray();
+
+    if (!data.length) {
+      statusCode = StatusCodes.NOT_FOUND;
```

Comment:
I see StatusCodes too many times... let's deconstruct it in the beginning of the file like so:

`const { NOT_FOUND, OK, BAD_REQUEST, GATEWAY_TIMEOUT } = StatusCodes`